### PR TITLE
[BB-07] Animations and effects on the website

### PR DESCRIPTION
**Pull request by PyyLog**
YouTrack: [BB-07](https://irisrobotics.youtrack.cloud/issue/PMI-62)

### CODE REVIEW

**Comment by PyyLog** [comment]
I think you can do a test with the Gradio space opened, to see the loader behaviour. Normally, it should disappear before the output is displayed !

File: src/App.css [hunk 1]
```diff
@@ -111,6 +115,110 @@
...
+.loader-container {
+	display: none;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	z-index: 999;
+	top: 50%;
+	left: 50%;
+	transform: translate(-50%, -50%);
+	background: #000;
```
Comment (PhantHive) [inline src/App.css]: I think without this background it would be more sick! ;)

**Review by PhantHive — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (PyyLog) [reply src/App.css]: Got it !

File: src/main.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import App from './App.tsx';
-import './index.css';
+import './styles/index.css';
```
Comment (PhantHive) [inline src/main.tsx:4]: You already imported this in the index

**Review by PhantHive — CHANGES_REQUESTED** [review]

File: src/main.tsx (hunk 2 above)
Comment (PyyLog) [reply src/main.tsx:4]: Can you review the PR please ?

Comment (PyyLog) [reply src/main.tsx:4]: If I delete this line, the base style in index.css is not respected ... so I keep it in the file.

**Comment by PyyLog** [comment]
It's all ok for this PR. Can you review please ?

File: src/main.tsx (hunk 2 above)
Comment (PhantHive) [reply src/main.tsx:4]: hmmm that should not be the case, it mean we are missing the import process we shoudl'nt have duplicate style imports

**Comment by PhantHive** [comment]
> It's all ok for this PR. Can you review please ?

I can review it tomorrow

**Comment by PyyLog** [comment]
> > It's all ok for this PR. Can you review please ?
> 
> I can review it tomorrow

Ok thanks !

File: src/main.tsx (hunk 2 above)
Comment (PhantHive) [reply src/main.tsx:4]: nice i saw you fixed it in [this commit](https://github.com/ipsa-scrypt/big-brain/commit/78b0694bcb116fc0d28323a18db040a1a9ef2094) 
All good for me!

**Review by PhantHive — APPROVED** [review]
giphy

**Pull request merged**